src/nidmm/templates/session.py/get_cal_date_and_time.py.mako

```diff
@@ -0,0 +1,27 @@
+<%page args="f, config"/>\
+    def get_cal_date_and_time(self, cal_type):
+        '''get_cal_date_and_time
+
+        Returns the date and time of the last calibration performed.
```

Comment:
it's annoying that the same info is present here and in get_cal_date_and_time.rst.mako. Is there no better way to single source it?

Reply:
An alternative would be to add new functions in function_addons and then use method_templates. You would still need specialized templates though because these handwritten functions are so specific it's not work torquing around the code generator to support them that way.

Reply:
I think it's worth it, especially if you can inherit some information from the original function you are wrapping (by assigning metadata from the original dictionary).